tasks.py

```diff
@@ -22,6 +24,11 @@ def genspider(ctx, name, domain):
     test_filename = _gen_tests(name, domain)
     print('Created {0}'.format(test_filename))
 
+    if start_urls:
+        html_filenames = _gen_html(name, start_urls)
```

Comment:
This is mostly my own curiosity -- does `invoke` automagically expand a comma separated list into a a Python list when passed as an argument?